txclib/project.py

```diff
@@ -108,32 +108,40 @@ def getset_host_credentials(self, host, username=None, password=None,
 
         save = False
         config_username, config_password = None, None
+        if 'TX_TOKEN' in os.environ:
+            # User api info from environment variable
+            password = os.environ['TX_TOKEN']
+            username = 'api'
+            """We need to check if hostname info exists in the .transifexrc file
+            If not, it means that this is the first time this function runs, so
+            we need to create its data.
+            """
+            save = self._update_file_host(host)
+            if save:
+                self.save()
+            if token:
```

Comment:
@Diontsoumas it think an info message in any case would be useful here. You could have have two logger statements like this:
```
logger.info("Using TX_TOKEN environment variable. Credentials in .transifexrc won't be updated")
if token:
    logger.warning("The --token argument will be ignored")
```
@samuelchan22 what do you think?

Reply:
@SofiaMargariti there is already a warning for the second case "A token has been found in TX_TOKEN env variable and --token argument. The latter will be ignored and no data will be saved in the .transifexrc file". I will update the PR once @samuelchan22 gives feedback for both copies.

Reply:
Sorry I missed the notification yesterday. Here are some tweaks:

> Using TX_TOKEN environment variable. Credentials won't be saved to .transifexrc.

> There is a token in the TX_TOKEN env variable and the --token argument. The latter will be ignored and no credentials will be saved to the .transifexrc file.

Wasn't sure if we should "won't be saved to" or "won't be updated in" — I guess we should go with whichever one is more accurate.